templates/server/puppetserver/conf.d/webserver.conf.erb

```diff
@@ -0,0 +1,15 @@
+webserver: {
+    access-log-config = <%= scope.lookupvar('puppet::server_puppetserver_dir') %>/request-logging.xml
+    client-auth       = want
+    ssl-host          = 0.0.0.0
+    ssl-port          = <%= scope.lookupvar('puppet::server_port') %>
+    ssl-cert          = <%= scope.lookupvar('puppet::server_ssl_dir') %>/certs/<%= @fqdn.downcase %>.pem
+    ssl-key           = <%= scope.lookupvar('puppet::server_ssl_dir') %>/private_keys/<%= @fqdn.downcase %>.pem
+<%- if scope.lookupvar('puppet::server_ca') -%>
+    ssl-ca-cert       = <%= scope.lookupvar('puppet::server_ssl_dir') %>/ca/ca_crt.pem
+    ssl-cert-chain    = <%= scope.lookupvar('puppet::server_ssl_dir') %>/ca/ca_crt.pem
+    ssl-crl-path      = <%= scope.lookupvar('puppet::server_ssl_dir') %>/ca/ca_crl.pem
+<%- else -%>
+    ssl-ca-cert       = <%= scope.lookupvar('puppet::server_ssl_dir') %>/certs/ca.pem
```

Comment:
never mind :bow:, in the old style puppet.conf, things like `$certname` are used then. Instead of `fqdn.downcase`, `<%= scope.lookupvar("puppet::server_certname") %>` could be used, although I also see that `fqdn` is used for some things already in teh current code, so I think this can be left as-is for the moment and we can think about how to handle this after this PR is merged.